examples/lock-app/qpg/src/AppTask.cpp

```diff
@@ -105,18 +105,17 @@ void OnTriggerIdentifyEffect(Identify * identify)
 {
     sIdentifyEffect = identify->mCurrentEffectIdentifier;
 
-    if (identify->mCurrentEffectIdentifier == Clusters::Identify::EffectIdentifierEnum::kChannelChange)
+    if (identify->mEffectVariant != Clusters::Identify::EffectVariantEnum::kDefault)
```

Comment:
Again QPG

Reply:
it won't cause them any harm and it was a quick change